Bullwinkle.class.nut

```diff
@@ -561,9 +595,7 @@ class Bullwinkle.Package {
             local d = date();
             return format("%d.%06d", d.time, d.usec);
         } else {
-            local d = math.abs(hardware.micros());
-            return format("%d.%06d", d/1000000, d%1000000);
+            return format("%d.%06d", time(), hardware.micros());  //this can be a bit of an ugly _ts but it allows us to calculate latencies up to 36 minutes long...
```

Comment:
True - it is not a subvalue of time and array makes more since.  However, I was trying to keep consistency between the agent and device code implementations as much as possible (package._ts is always a string as opposed to a string on the agent and an array on the device).

Timeouts work off of the integer time() in seconds, and ```getLatency``` knows what is going on, but the implementation is confusing and ugly.

Since ._ts is a "private" _property, people shouldn't be depending on it and we can likely move to an array?